tests/foreman/virtwho/utils.py

```diff
@@ -72,13 +72,13 @@ def get_guest_info():
     return guest_name, guest_uuid
 
 
-def runcmd(cmd, system=None, timeout=None, output_format='plain'):
+def runcmd(cmd, system=None, timeout=None, output_format=base):
```

Comment:
Miss quotes here

Reply:
@hkx303 , sorry, fixed. 

I notice also:
robottelo/helpers.py has output_format='plain'

Thank you

Reply:
yes, and Travis failed due to 
```
            ret = ssh.execute_command(
                'ls -la', connection, output_format='plain')
```
in [robottelo_testing](https://github.com/SatelliteQE/robottelo/blob/master/tests/robottelo/test_ssh.py#L247)

Reply:
OK, found it:  `robottelo/tests/robottelo/test_ssh.py`